config-linux.md

```diff
@@ -197,45 +197,62 @@ Network devices have their own network namespace and a set of operations distinc
 
 This schema focuses solely on moving existing network devices identified by name into the container namespace. It does not cover the complexities of network device creation or network configuration, such as IP address assignment, routing, and DNS setup.
 
+Only one IP address is allowed for initial network device setup. This restriction is intended to reduce complexity during the container creation and to mitigate the risk during container start of IP address duplication. Adding additional IP addresses to the interface can be handled through post-configuration or by the container application itself.
```

Comment:
IPv6 unfortunately uses link-local addresses in addition to any other number of addresses for prefixes that are announced on link, which the host/container/entity is supposed to properly autoconfigure. While this is not an address delegation protocol, maybe at least IPv6 should be given the option of >1 addresses?

Reply:
Actually, we do have a need for accelerators using more than one IPv4 address also.

Reply:
in any case, if we want to pluralize at the beginning, then we should pick a max number, leaving an unbounded number of IP addresses will complicate the implementation and will open a way of users to shoot themselves in the foot

Reply:
changing structure member from single string to array of strings is breaking change. IMHO, it is cheaper to do from beginning plural notation for addresses, and from implementation it doesn't matter that much if it calls only once netlink add address and checks error, vs. doing same iterating over array.

Reply:
fair, let me move to an array as I'm not strong on this and seems to be the only friction point

Reply:
@kad @pfl pluralized the field `address` to be `addresses` and allow an array of IPs

Reply:
That will work very well for us, thanks!